mantle/kola/tests/rhcos/luks.go

```diff
@@ -35,6 +38,84 @@ func init() {
 			}
 		}`),
 	})
+	register.RegisterTest(&register.Test{
+		Run:                  luksTangTest,
+		ClusterSize:          0,
+		Name:                 `rhcos.luks.tang`,
+		Flags:                []register.Flag{},
+		Distros:              []string{"rhcos"},
+		Platforms:            []string{"qemu-unpriv"},
+		ExcludeArchitectures: []string{"s390x", "ppc64le"}, // no TPM support for s390x, ppc64le in qemu
+	})
+	register.RegisterTest(&register.Test{
+		Run:                  luksSSST1Test,
+		ClusterSize:          0,
+		Name:                 `rhcos.luks.sss.t1`,
+		Flags:                []register.Flag{},
+		Distros:              []string{"rhcos"},
+		Platforms:            []string{"qemu-unpriv"},
+		ExcludeArchitectures: []string{"s390x", "ppc64le"}, // no TPM support for s390x, ppc64le in qemu
+	})
+	register.RegisterTest(&register.Test{
+		Run:                  luksSSST2Test,
+		ClusterSize:          0,
+		Name:                 `rhcos.luks.sss.t2`,
+		Flags:                []register.Flag{},
+		Distros:              []string{"rhcos"},
+		Platforms:            []string{"qemu-unpriv"},
+		ExcludeArchitectures: []string{"s390x", "ppc64le"}, // no TPM support for s390x, ppc64le in qemu
+	})
+}
+
+func getEncodedTangPin(c cluster.TestCluster) string {
+	return b64Encode(getTangPin(c))
+}
+
+func startTang(c cluster.TestCluster) {
+	if err := exec.Command("sudo", "xinetd").Run(); err != nil {
```

Comment:
You don't have to do this right now but I imagine it'd work to run tang as non-root, not even clear to me we need xinetd.  But again that can be enhanced in the future.